src/cpp/src/utils.hpp

```diff
@@ -50,6 +51,7 @@ Config from_config_json_if_exists(const std::filesystem::path& models_path, cons
 }
 
 ov::genai::StreamerVariant get_streamer_from_map(const ov::AnyMap& config_map);
+ov::genai::ChunkStreamerVariant get_chunk_streamer_from_map(const ov::AnyMap& config_map);
```

Comment:
it's whisper specific entity. Maybe we can move it to whisper files? the same in other places like py_utils.hpp

these utils are supposed to be generic ones

Reply:
Moved all chunk streamer related code to whisper files.